docs/testing.md

```diff
@@ -59,9 +59,11 @@ Acceptance tests will be written for all command handlers and will be run as par
 
 You can run these tools manually with composer:
 
+> **Notice**: The behat tests sometimes gives a false positive, if the tests fail retry the tests before troubleshooting
```

Comment:
Muchios gracias for adding this warning!

Proposal to make this sentence more specific:
old: `, if the tests fail retry the tests before troubleshooting`
new: `. If the <<inser test name here>> test fails, please re-run the behat tests before troubleshooting`